### PR TITLE
Issue #1152: Transaction Advisors case study & assets

### PR DESCRIPTION
**Pull request by dmurphy1**
@AnneTee - this PR is for the Transaction Advisors case study. It creates the case study, creates 3 new tags (multisite, migration, payment-processing) and includes image assets.

For the images, I was thinking:
- **Hero image**: just add the standard overlay and opacity to `transaction_advisors_hero.png`
- **Objective image**: The `transaction_advisors_desktop.png` and `transaction_advisors_mobile.png` images together, similar to the [Hunter results](http://savaslabs.com/work/hunter-boots-case-study/)
- **Approach image**: The `satellitefinance_desktop.png` and `satellitefinance_mobile.png` images together, similar to the [Hunter results](http://savaslabs.com/work/hunter-boots-case-study/)
- **Results image**: The `telecomfinance_desktop.png` and `telecomfinance_mobile.png` images together, similar to the [Hunter results](http://savaslabs.com/work/hunter-boots-case-study/)

I'm open to anything you think looks good though. In addition to formatting the image assets, could you do a quick spelling/grammar spotcheck in case I missed anything?

Thanks!!

### CODE REVIEW

**Comment by AnneTee** [comment]
@dmurphy1 Just opened a PR against your branch with the images! https://github.com/dmurphy1/savaslabs.github.io/pull/1

**Comment by dmurphy1** [comment]
Thanks @AnneTee - I think it looks good. Can we merge into master or should we get @chrisarusso's sign off first?

**Comment by AnneTee** [comment]
@dmurphy1 I say we merge, @chrisarusso please let us know if you have any updates. ✨

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h1 class="home__cover__heading">Savas Labs</h1><hr>
       <!-- Most recent case studies. -->
       <div class="case-studies">
         <h2>Recent work</h2>
-        {% for item in site.case-studies limit 3 %}
+        {% for item in site.case-studies limit:3 %}
```
Comment (chrisarusso) [inline index.html:24]: Should this line have gotten incorporated @AnneTee / @dmurphy1 ?

Comment (AnneTee) [inline index.html:24]: @chrisarusso Yes, the colon is required. I had the syntax wrong before.

We're going to have to update this once the Pauli Murray case study is up to choose the order; this is a temporary measure.

**Pull request merged**